docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-conditions-autodiscover.asciidoc

```diff
@@ -0,0 +1,318 @@
+[[conditions-based-autodiscover]]
+= Conditions based autodiscover
+
+You can define autodiscover conditions in each input to allow {agent} to automatically identify Pods and start monitoring them using predefined integrations. You can find more about these inputs and their configurations <<elastic-agent-input-configuration,here>>.
+
+== Example: Target pods by host and label
```

Comment:
```suggestion
== Example: Target Pods by host and label
```
I'm only going to mark this once, but I believe that Kubernetes always capitalizes Pod in their docs. Apply this change globally.

Reply:
What do we mean here host and label?
The condition is just `condition: ${kubernetes.labels.app} == 'redis'`.